config/prod/s3-ingestion.yaml

```diff
@@ -0,0 +1,18 @@
+# NOTE:
+# This is the first bucket in the ingress pipeline for RECOVER project.
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-ingestion-prod"
+stack_tags:
+  OwnerEmail: 'synapse-service-sysbio-recover-data-storage-01@sagebase.org'
```

Comment:
I think based on this comment: https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/566#discussion_r1142612670
and also from the template, `CostCenter` and `OwnerEmail` aren't available in the template and should be removed

Reply:
Excellent - I removed `OwnerEmail` but `CostCenter is still required